deploy/operator.yaml

```diff
@@ -16,7 +16,7 @@ spec:
       containers:
         - name: integreatly-operator
           # Replace this with the built image name
-          image: quay.io/integreatly/integreatly-operator:v1.8.1
+          image: quay.io/integreatly/integreatly-operator:v1.8.2
```

Comment:
The manifest is still pointing at v1.8.0 , is there a version gone missing somewhere https://github.com/integr8ly/manifests/blob/master/Makefile#L2

Reply:
maybe, I'll catch it up when I PR

Reply:
:+1: